### PR TITLE
Update deploy.yml

### PR DESCRIPTION
**Pull request by inverse**
Leading on from #117 I noticed the image hasn't been updated for a while so thought getting this in sync might be a good thing.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -129,3 +129,14 @@ jobs:
...
+
+  deploy-dockerhub:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Publish to Dockerhub
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: ivandokov/phockup
+          username: ${{ secrets.DOCKER_USERNAME }}
```
Comment (inverse) [inline .github/workflows/deploy.yml:141]: You'll need to set these secrets first though.

**Comment by ivandokov** [comment]
Not sure how I forgot to do this. Thank you!

**Comment by ivandokov** [comment]
Unfortunatelly this didn't quite work the first time https://github.com/ivandokov/phockup/runs/3394912765?check_suite_focus=true

**Comment by inverse** [comment]
@ivandokov  Sorry 🙈 

I forgot to include one step before.

```
    - uses: actions/checkout@master
```

**Pull request merged**